RULES.md

```diff
@@ -1,536 +1,1037 @@
+|-------------------------------------------------------------------------	|--------------------------------------------------------	|--------------------------------------------------------------|
+| [`IOError`](#IOError)                                                   	| Error in IO operation.                                 	| i_o_error                                                    |
+| [`RuntimeExceptionInLoaderError`](#RuntimeExceptionInLoaderError)       	| RuntimeException while loading GTFS dataset in memory. 	| runtime_exception_in_loader_error                            |
+| [`RuntimeExceptionInValidatorError`](#RuntimeExceptionInValidatorError) 	| RuntimeException while validating GTFS archive.        	| runtime_exception_in_validator_error                         |
+| [`ThreadExecutionError`](#ThreadExecutionError)                         	| ExecutionException during multithreaded validation     	| thread_execution_error                                       |
+| [`URISyntaxError`](#URISyntaxError)                                     	| A string could not be parsed as a URI reference.       	| u_r_i_syntax_error                                           |
 
-## Notices
+## Errors
 
-### Errors
+<a name="BlockTripsWithOverlappingStopTimesNotice"/>
 
-#### BlockTripsWithOverlappingStopTimesNotice
+### BlockTripsWithOverlappingStopTimesNotice
 
 Trips with the same block id have overlapping stop times.
-
-##### References:
+<details>
+
+#### Notice fields description
+| Field name      	| Description                               	            | Type   	|
+|-----------------	|---------------------------------------------------------	|--------	|
+| `csvRowNumberA` 	| The row number from `trips.txt` of the first faulty trip. | Long   	|
+| `tripIdA`       	| The id of first faulty trip.                           	| String 	|
+| `serviceIdA`    	| The service id of the first faulty trip.                 	| String 	|
+| `csvRowNumberB` 	| The row number from `trips.txt` of the second faulty trip.| Long   	|
+| `tripIdB`       	| The id of the other faulty trip.             	            | String 	|
+| `serviceIdB`    	| The service id of the other faulty trip.     	            | String 	|
+| `blockId`       	| The `trips.block_id` of the overlapping trip.	            | String 	|
+| `intersection`  	| The overlapping period.                      	            | Date   	|
+
+#### Affected files
+* [`stops.txt`](http://gtfs.org/reference/static#stopstxt)
+* [`trips.txt`](http://gtfs.org/reference/static#tripstxt)
+
+#### References
 * [Original Python validator implementation](https://github.com/google/transitfeed)
+</details>
 
 <a name="CsvParsingFailedNotice"/>
 
-#### CsvParsingFailedNotice
+### CsvParsingFailedNotice
 
 Parsing of a CSV file failed. One common case of the problem is when a cell value contains more than 4096 characters.
 
+<details>
+
+#### Notice fields description
```

Comment:
Maybe `#### Notice JSON fields`? Do these appear anywhere else (outside of the JSON) that a producer/consumer might need to know them?

Reply:
In the HTML output it is in a table, I think it's best not to use JSON here for users that only use the HTML output to see the results.
The HTML output:
![Screen Shot 2022-05-17 at 9 32 56 AM](https://user-images.githubusercontent.com/63653518/168823116-fc9d6c81-756e-4f97-9da5-3e7a885616aa.png)

The documentation:

![Screen Shot 2022-05-17 at 9 58 07 AM](https://user-images.githubusercontent.com/63653518/168828554-3b58204d-ed6a-4024-9c4f-71235ee9b1b1.png)